Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -135,8 +137,22 @@ private void addSmartDashboardWidgets( )
     SmartDashboard.putData("AutoDrivePathForward2", new AutoDrivePath(m_swerve, "forward2m", true));
     SmartDashboard.putData("AutoDrivePathBackward2", new AutoDrivePath(m_swerve, "backward2m", true));
 
-    // On-the-fly path generation buttons
-    SmartDashboard.putData("ResetOdometryToLimelight", new ResetOdometryToLimelight(m_swerve, m_vision, 0));
```

Comment:
Maybe I missed it, but I didn't see this line (ResetOdometeryToLimelight) added back in anywhere. Is it gone now?